templates/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<h1>Login</h1>
+<form method="post">
+    {{ form.csrf_token }}
+    <p>
+        {{ form.first_name.label }}<br>
+        {{ form.first_name(size=32) }}
+    </p>
+    <p>
+        {{ form.last_name.label }}<br>
+        {{ form.last_name(size=32) }}
+    </p>
+    <p>
+        {{ form.email.label }}<br>
+        {{ form.email(size=32) }}
+    </p>
+    <p>
+        {{ form.password.label }}<br>
+        {{ form.password(size=32) }}
+    </p>
+    <p>
+        {{ form.confirm_password.label }}<br>
+        {{ form.confirm_password(size=32) }}
+    </p>
+    <p>
+        <input type="submit" value="Отправить">
+    </p>
```

Comment:
А это нужно оптимизировать. Что если форма будет иметь более 5 полей? 
```suggestion
    {% for obj in form %}
    {% if obj.name == "csrf_token" %}
    {{ obj }}
    {% else %}
    <p>{{ obj.label }}: {{ obj }}</p>
            {% for error in obj.errors %}
                <span style="color: red;">{{ error }}</span><br>
            {% endfor %}
    {% endif %}
    {% endfor %}
```
Причем строки
```html
            {% for error in obj.errors %}
                <span style="color: red;">{{ error }}</span><br>
            {% endfor %}
```
будут повторять flash сообщение об ошибке. Тут вы можете решить, что лучше: либо выводить сообщение тут, либо выше.